doc/gc.rst

```diff
@@ -19,11 +19,8 @@ This document describes how the GC works and how to tune it for
 
 The basic algorithm is *Deferred Reference Counting* with cycle detection.
 References on the stack are not counted for better performance (and easier C
-code generation). Cycle detection is currently done by a simple mark&sweep
-GC that has to scan the full (thread local heap). ``--gc:v2`` replaces this
-with an incremental mark and sweep. That it is not production ready yet,
-however.
-
+code generation). Cycle detection is currently done by a simple Mark&Sweep
+GC that has to scan the full (thread local heap).
 
 The GC is only triggered in a memory allocation operation. It is not triggered
```

Comment:
That should also (maybe) be updated, because with --gc:arc that's not really the case

Reply:
Does it really needs a dedicated introduction section ?.

I think thats all redundant info, and if its not then should be integrated on each garbage collector description, for DRY.
Also it wont apply for ARC nor Manual anyways.